src/components/common/PaginatedTxns/index.tsx

```diff
@@ -9,18 +10,17 @@ import InfiniteScroll from '../InfiniteScroll'
 import SkeletonTxList from './SkeletonTxList'
 import BatchExecuteButton from '@/components/transactions/BatchExecuteButton'
 import TxFilterButton from '@/components/transactions/TxFilterButton'
-import { TxFilter, useTxFilter } from '@/utils/tx-history-filter'
-import { isTransactionListItem } from '@/utils/transaction-guards'
-import type { TransactionListPage } from '@gnosis.pm/safe-react-gateway-sdk'
+import { type TxFilter, useTxFilter } from '@/utils/tx-history-filter'
 import { BatchExecuteHoverProvider } from '@/components/transactions/BatchExecuteButton/BatchExecuteHoverProvider'
+import { isTransactionListItem } from '@/utils/transaction-guards'
 
 const NoQueuedTxns = () => (
   <Box mt="5vh">
     <PagePlaceholder imageUrl="/images/no-transactions.svg" text="Queued transactions will appear here" />
   </Box>
 )
 
-const getResultCount = (filter: TxFilter, page: TransactionListPage) => {
+const getFilterResultCount = (filter: TxFilter, page: TransactionListPage) => {
```

Comment:
![Screenshot 2022-09-15 at 12 59 15](https://user-images.githubusercontent.com/32431609/190387483-e4dda706-3fc3-4b85-a377-d1bca348e14c.png)
 is not necessarily true if the filtered transactions are more than 1 page

Reply:
Yeah, it just refers to the first page, hence the `>` sign. I haven't changed anything with this component.

Reply:
Oh! The `>` was not obvious at first sight. 

>  I haven't changed anything with this component.

Yeah ik, was just a small remark